VitvitskyAkim/HW7/task7_3.py

```diff
@@ -0,0 +1,7 @@
+def number_of_characters(word):
```

Comment:
good idea
look at this version of code:
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result